src/components/header.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from "gatsby"
+import PropTypes from "prop-types"
+import React from "react"
+
+const Header = ({ siteTitle }) => (
```

Comment:
So this is the part that is React.

You haven't learned about variables or functions yet. This is kinda like a combination of both, but instead of a variable, a function is being stored as a constant.

The function is returning a React component, which wraps HTML and CSS.